Cargo.toml

```diff
@@ -19,6 +19,8 @@ rustdoc-args = ["--cfg", "docsrs"]
 default = [ "simple_http", "simple_tcp" ]
 # A bare-minimum HTTP transport.
 simple_http = [ "base64" ]
+# A transport that uses `minreq` as the HTTP client.
+simple_minreq = [ "base64", "minreq" ]
```

Comment:
In d20ba2eafcaa3cd5245f5a548cbcbb6d14c13432:

This should be called `minreq_http` since `minreq` is the implementation, not the protocol.